Userland/Libraries/LibWeb/Page/EventHandler.cpp

```diff
@@ -75,6 +75,10 @@ static Gfx::StandardCursor cursor_css_to_gfx(Optional<CSS::Cursor> cursor)
     case CSS::Cursor::NwseResize:
         return Gfx::StandardCursor::ResizeDiagonalTLBR;
 
+    case CSS::Cursor::ZoomIn:
+    case CSS::Cursor::ZoomOut:
+        return Gfx::StandardCursor::Zoom;
+
```

Comment:
Do me a favor and get rid of the newlines from line 53 onwards instead, who did this lol